README.md

```diff
@@ -1,49 +1,35 @@
 # react-wp-scripts
 
-A wrapper for create-react-app's `react-scripts` to allow seamless usage of scripts and styles served from `webpack-dev-server` while developing a theme or plugin.
+A wrapper for create-react-app's [`react-scripts`](https://github.com/facebookincubator/create-react-app/tree/master/packages/react-scripts) to allow seamless usage of scripts and styles served from `webpack-dev-server` while developing a theme or plugin.
 
 **Important Note**: This project is brand new, and largely untested. We recommend using it as a learning tool rather than depending on it for critical development work.
 
 ## Installation & Usage
 
-From the root directory of your `create-react-app`-generated project, run
+Run `create-react-app --scripts-version react-wp-scripts --php-namespace="Your_Namespace" /path/to/your/project/folder` to generate a new create-react-app project configured to use these custom scripts.
```

Comment:
Actually I didn't realize initially but these docs have lagged, too -- the `--php-namespace` argument's been removed entirely in favor of parsing the project name. I kind of liked making that pluggable, but at minimum we should explain that the namespace does NOT default to ReactWPScripts unless the packageJSON cannot be parsed, in which case the init is likely to fail anyway.

I honestly don't know that I like auto-generating the namespace from the project; @rmccue why did you feel it was preferable to using an opinionated default like ReactWPScripts? If I have a project `H2`, I wouldn't want the namespace for my script-loader file to be `H2`.

Reply:
@kadamwhite see discussion on #21. I initially introduced a simple _guard_, because I see it pretty much like you. The functions are part of the `react-wp-script` package, and not of _my_ app, whatever it is that I am using `react-wp-scripts` with.

@rmccue then mentioned the possibility to allow for multiple (potentially **different**) version of the functions being used. Not sure if this is to be aimed for at all, but that was where our ideas/approaches diverged.

Reply:
> If I have a project `H2`, I wouldn't want the namespace for my script-loader file to be `H2`.

On the contrary, that's exactly what I want. After init, the loader file is part of **my** code, not a library being loaded in. It should match that.